### PR TITLE
feat(SPV-938): upgrade to go-sdk

### PR DESCRIPTION
**Pull request by yarex-4chain**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/go-broadcast-client/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/go-broadcast-client/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [x] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: broadcast/internal/arc/arc_submit_tx.go [hunk 1]
```diff
@@ -16,6 +16,10 @@ import (
 	"github.com/libsv/go-bt/v2"
 )
 
+// TODO: NewTxFromString - No direct replacement
+// TODO: NewTxFromBytes - not found
+// TODO: Input - not found (TransactionInput?)
```
Comment (wregulski) [inline broadcast/internal/arc/arc_submit_tx.go:21]: Yes, it's the same that TransactionInput

File: examples/query_transaction/query_tx.go [hunk 2]
```diff
@@ -16,7 +16,7 @@ func main() {
 	token := "mainnet_06770f425eb00298839a24a49cbdc02c"
 	apiURL := "https://arc.taal.com"
 	deploymentID := "broadcast-client-example"
-	txID := "469dd0f63fe4b3fe972dc72d28057e931abd69d8dfc85bf6e623efa41d50ef73"
+	txID := "c67d504ea27eb225655bdb1f29c3a1a8f2056d675f3a7b17e91638efa30e1665"
```
Comment (dorzepowski) [inline examples/query_transaction/query_tx.go:19]: why you changed it?

File: go.mod [hunk 3]
```diff
@@ -1,6 +1,8 @@
 module github.com/bitcoin-sv/go-broadcast-client
 
-go 1.21.5
+go 1.22
+
+toolchain go1.22.5
 
 require (
 	github.com/libsv/go-bt/v2 v2.2.5
```
Comment (dorzepowski) [inline go.mod:8]: We still use this somewhere, and I assume that we want to get rid of it also.

**Review by dorzepowski — CHANGES_REQUESTED** [review]

File: go.mod (hunk 3 above)
Comment (wregulski) [reply go.mod]: i got rid of that dependency from go.mod

File: examples/query_transaction/query_tx.go (hunk 2 above)
Comment (wregulski) [reply examples/query_transaction/query_tx.go]: unfortunately the previous one started throwing 404 not found - updated to a working, fresh one

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**